packages/client-adapters/tests/adapters/lexicons/adapter.test.js

```diff
@@ -42,6 +42,7 @@ describe('lexicons/adapter.test.js', () => {
   beforeEach(async () => {
     ClientAdapters.init()
     testLangID = Constants.LANG_GREEK

```

Comment:
do we need testLangID anymore?

Reply:
Yes, unfortunately. Client Adapters require a language ID as a parameter, so the language ID must be here: https://github.com/alpheios-project/alpheios-core/blob/dev-annotations-next/packages/client-adapters/tests/adapters/lexicons/adapter.test.js#L53